kymatio/frontend/tensorflow_frontend.py

```diff
@@ -0,0 +1,6 @@
+class ScatteringTensorflow(object):
```

Comment:
Is there nothing to inherit from? Maybe a `tf.keras` object?

Reply:
`tf.keras.layers.Layer` with `trainable=False` and `dynamic=False`

https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer